tests/MongoDB.Driver.Tests/Linq/IntegrationTestBase.cs

```diff
@@ -81,7 +81,9 @@ private void InsertFirst()
                                     {
                                         D = "Delilah"
                                     }
-                            }
+                            },
+                            N = 1,
+                            Y = new List<E> { new E { F = 101 }, new V { W = 102 } }
```

Comment:
This class is getting out of control. Too many different tests depend on it and that's why it has way too many fields.

Reply:
Completely agree. Maybe we can refactor it by class generic param + common(somehow) logic for collection initializing?